src/base-element.js

```diff
@@ -40,16 +40,11 @@ import {isArray, toWin} from './types';
  *           || createdCallback
```

Comment:
Please delete this.

Reply:
It is unclear to me if the intention is to document CE callbacks as wel.

Reply:
Done